src/Microsoft.Framework.Configuration.Abstractions/IConfiguration.cs

```diff
@@ -9,23 +9,8 @@ public interface IConfiguration
     {
         string this[string key] { get; set; }
 
-        /// <summary>
-        /// 
-        /// </summary>
-        /// <param name="key">A case insensitive name.</param>
-        /// <returns>The value associated with the given key, or null if none is found.</returns>
-        string Get(string key);
```

Comment:
Can you please make sure we have an issue to track that we need to fill our XML comments?